_posts/2018-02-21-symfony-route-test.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Testing the new symfony router
+author: Georgos Landin Xanthopoulos
+date: '2018-02-02 15:06:47 +0200'
+header:
+  teaser: images/posts/teaser2.jpg
+  caption: "Photo credit: [**The Preiser Project**](https://www.flickr.com/photos/thepreiserproject/)"
+categories:
+- Symfony
+---
+<b> How much faster is the new Symfony router? 
+We ran some tests to find out.</b>
+
+## An improved router
+
+A few days ago we received the fantastic news that the Symfony router had been improved significantly, of course we wanted to see how
+the changes would affect our application, so we decided to test it against our own routes. 
+
+If you somehow missed the exciting news you can read this [post](https://medium.com/@nicolas.grekas/making-symfonys-router-77-7x-faster-1-2-958e3754f0e1) from Nicolas Grekas. 
```

Comment:
s/exciting news/announcements/


Here is the official announcement:
https://symfony.com/blog/new-in-symfony-4-1-fastest-php-router

The medium post is a deep dive